service.py

```diff
@@ -1,107 +1,142 @@
 from concurrent import futures
 import grpc
-import time
 import utility
-import io
 import hwsc_file_transaction_svc_pb2
 import hwsc_file_transaction_svc_pb2_grpc
+from enum import Enum
+
+
+state = 0
+
+
+class State(Enum):
+    """A current state class of file transaction service"""
+    AVAILABLE = 0
+    UNAVAILABLE = 1
+
 
 class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
-     """A FileTransactionService class contains servicer and corresponding functionalites."""
-     def __init__(self):
-
-         class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
-             """A servicer class contains functionalities for file service."""
-             def __init__(self):
-                 pass
-
-             #TODO
-             def GetStatus(self, request, context):
-                 """Get and return status to the user."""
-                 print("Get Status")
-
-             #TODO
-             def DownloadZippedFiles(self, request_iterator, context):
-                 """Download zipped files from azrue blob storage."""
-                 if request_iterator.name:
-                     return utility.download_chunk(self.tmp_file_name)
-
-             def UploadFile(self, request_iterator, context):
-                 """Upload a file to the azure blob storage."""
-                 print("[INFO] Requesting UploadFile service")
-                 #TODO
-                 #May implement try-catch block
-
-                 d = utility.get_property(request_iterator)
-
-                 type = utility.get_file_type(d["f_name"])
-                 is_uuid_valid = utility.verify_uuid(d["uuid"])
-
-                 if is_uuid_valid:
-                     has_folder = utility.find_folder(d["uuid"], type)
-                     get_url = utility.upload_file_to_azure(d["stream"], has_folder, d["uuid"], d["f_name"])
-
-                     #TODO
-                     #write a function to validate the url, like a regex check
-                     if get_url != "":
+    """A FileTransactionService class contains servicer and corresponding functionalites."""
+
+    def __init__(self):
+
+        class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
+            """A servicer class contains functionalities for file service."""
+
+            def __init__(self):
+                pass
+
+            def GetStatus(self, request, context):
+                """Return the status of the service"""
+                print("[INFO] Requesting GetStatus service")
+                global state
+                state = State.AVAILABLE.value
+                available = grpc.StatusCode.OK.value[0]
+

```

Comment:
`GetStatus` should not mutate the `state`. It only checks the `state`. 

`global state` this line is good enough

line 33 and line 34 are NOT needed

Reply:
if i delete line 33, 34, the if statement will look like this, does it ok ?

if grpc.StatusCode.OK.value[0] != state.value

Reply:
` if state != State.AVAILABLE:`